cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,46 @@
-describe('', () => {
-  before(() => {
+/// <reference types='cypress' />
 
+let user;
+let article;
+
+describe('Conduit', () => {
+  beforeEach(() => {
+    cy.visit('');
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
+
+    cy.task('generateArticle').then((generateArticle) => {
+      article = generateArticle;
+    });
   });
 
-  it('', () => {
+  it('should login and create article', () => {
+    cy.login(user.email, user.username, user.password);
```

Comment:
The login function seems to be called with the username as the second parameter, but usually, the login function only requires an email and a password. Please, verify if the parameters are correct.